src/buttons/test/buttons.spec.js

```diff
@@ -328,12 +328,38 @@ describe('buttons', function() {
         $scope.uncheckable = false;
         var btns = compileButtons('<button ng-model="model" uib-btn-radio="1">click1</button><button ng-model="model" uib-btn-radio="2" uib-uncheckable="uncheckable">click2</button>', $scope);
         expect(btns.eq(0).attr('uncheckable')).toBeUndefined();
-        expect(btns.eq(0).attr('uncheckable')).toBeUndefined();
+        expect(btns.eq(1).attr('uncheckable')).toBeUndefined();
+
+        expect($scope.model).toBeUndefined();
+
+        btns.eq(0).click();
+        expect($scope.model).toEqual(1);
+
+        btns.eq(0).click();
```

Comment:
That is to assert that, with `uncheckable` not being set, clicking the same button twice is the same as clicking it once. Otherwise, it might just always be uncheckable irrespective of the state of `uncheckable`.